src/components/templates/Jazzbar/JazzTab/JazzTab.tsx

```diff
@@ -34,6 +26,14 @@ import { useVenueId } from "hooks/useVenueId";
 
 import { JAZZBAR_TABLES } from "./constants";
 
+// NOTE: This functionality will probably be returned in the nearest future.
+// import CallOutMessageForm from "components/molecules/CallOutMessageForm/CallOutMessageForm";
+import Room from "../components/JazzBarRoom";
+import JazzBarTableComponent from "../components/JazzBarTableComponent";
+import TableHeader from "components/molecules/TableHeader";
+import TablesUserList from "components/molecules/TablesUserList";
+import { BackButton } from "components/atoms/BackButton";
+
```

Comment:
Why was this block moved down here?

Reply:
They were not ordered correctly. I just moved them at the bottom. I also just noticed that `components/` are after `./components` I will change that as well.

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1492/commits/516bcef90616e538cbf0a8609a96d134bfbf4fcc